src/logger.cpp

```diff
@@ -36,6 +36,7 @@ struct Logger::LoggerImpl
     std::vector<size_t> mErrors;
     std::vector<size_t> mWarnings;
     std::vector<size_t> mHints;
+    std::vector<size_t> mInformations;
```

Comment:
Urgh.  Not a fan of the "informations" thing.  Maybe it could be a type of `LOG`? which would then give us `logCount()`, `mLogs` etc and a more natural language to describe it?

Reply:
Not keen on log. A log is something that I would use to output debug information, which is not what we are after here.

Reply:
agree log is not ideal - in the logger class log would usually mean the whole thing...to me at least. And I'm not too worried about "natural language" :) If anything I'd slightly prefer `INFO` over `INFORMATION`.

Reply:
Ok, if you would only slightly prefer `INFO` over `INFORMATION` then personally I would stick with `INFORMATION`, just for consistency (i.e. use full words rather than a short version of them).

@kerimoyle, up to you now... :)

Reply:
see #674.